### ISSUE TITLE
Off-by-one error when using `PRINT#` and `INPUT#` on random file

### ISSUE THREAD

**Issue by robhagemans** [robhagemans/pcbasic#32]
```
10 OPEN "r",1,"test"
20 PRINT #1,"Hello"
30 PUT 1,1
40 GET 1,1
50 INPUT #1,a$
60 PRINT a$
```
writes `ello`, should be `Hello`

**Comment by robhagemans** [comment]
Occurs both in 15.08 and 16.12

**Comment by robhagemans** [comment]
Added test `INPUTGET` for this issue

**Comment by robhagemans** [comment]
Fixed in 16.12 by ea13749

**Comment by robhagemans** [comment]
Fixed in 15.08 by d7774d4
